pkg/controllers/interfaces.go

```diff
@@ -18,3 +18,15 @@ type eventRecorder interface {
 type controllerManager interface {
 	ctrl.Manager
 }
+
+// used for mocks
+
+//goland:noinspection GoUnusedType
+type etcdRegistry interface {
+	registry.Registry
+}
+
+//goland:noinspection GoUnusedType
+type etcdContext interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `etcdContext` is unused